runtime/src/main/jni/ObjectManager.cpp

```diff
@@ -513,6 +517,18 @@ void ObjectManager::MarkReachableObjects(Isolate* isolate, const Local<Object>&
         } // for
 
     } // while
+
+    if (frame.check()) {
+        auto cls = fromJsInfo->ObjectClazz;
+        JEnv env;
+        jclass clsClazz = env.GetObjectClass(cls);
+        jmethodID methodId = env.GetMethodID(clsClazz, "getSimpleName", "()Ljava/lang/String;");
+        jstring className = (jstring) env.CallObjectMethod(cls, methodId);
+        const char* str = env.GetStringUTFChars(className, NULL);
```

Comment:
Lines 527, 528, 530 can be replaced with the following
`auto jsObjClassName = "MarkReachableObjects: " + ArgConverter::jstringToString(className);`